docs/built-in-models/linear.md

```diff
@@ -28,6 +28,9 @@ The `n_bits` parameter controls the bit-width of the inputs and weights of the l
 
 The `n_bits` parameter can be set to `8` or more bits for models with up to `300` input dimensions. When the input has more dimensions, `n_bits` must be reduced to `6-7`. All performance metrics are preserved down to `n_bits=6`, compared to the non-quantized float models from scikit-learn.
 
+The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to uniformize the distribution of the different features.
```

Comment:
- do we mention using **Scaler directly in the doc? I think it would be interesting to have code block directly with one Scaler example
- I think it would be interesting that also, we have the word 'normalization', such that it's searchable in the doc

Reply:
The goal here is really to make the user take a look at the notebook such that it's clear what the impact of it can be and  they can directly copy-paste the snippet from there.

Could make sense to also have a snippet here but as you want

Reply:
I think we need to convince them there is a big problem here, else they'll never go to the nb

Reply:
I don't think a code snippet will do that tbh

It might not be an issue for datasets where the features have more or less the same range.

Reply:
it's not to convince them there is a problem with the code snippet: it's to tell them how to not have the problem with code snippet, and to have Scaler and normalization somewhere in the doc